spring-cloud-stream-binder-kafka-streams/src/main/java/org/springframework/cloud/stream/binder/kafka/streams/KafkaStreamsBinderSupportAutoConfiguration.java

```diff
@@ -59,30 +63,58 @@ public KafkaStreamsBinderConfigurationProperties binderConfigurationProperties(K
 		return new KafkaStreamsBinderConfigurationProperties(kafkaProperties);
 	}
 
+	@Bean
+	public KafkaStreamsConfiguration kafkaStreamsConfiguration(KafkaStreamsBinderConfigurationProperties binderConfigurationProperties) {
+		return new KafkaStreamsConfiguration(
+				new HashMap<>(binderConfigurationProperties.getKafkaProperties().buildStreamsProperties()));
+	}
```

Comment:
Why are you overriding Boot's `@Bean` ? It has logic to use the `spring.application.name` if there is no ID.